src/contexts/Dashboard.js

```diff
@@ -0,0 +1,222 @@
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import weekOfYear from "dayjs/plugin/weekOfYear";
+import PropTypes from "prop-types";
+import React, {
+  useCallback,
+  useEffect,
+  useReducer,
+  useState,
+  createContext,
+  useContext,
+  useMemo,
+} from "react";
+
+import { GLOBAL_CHART } from "../apollo/queries";
```

Comment:
```suggestion
import { DASHBOARD_CHART } from "../apollo/queries";
```

Add below to queries

```
export const DASHBOARD_CHART = gql`
  query swaprDayDatas($startTime: Int!, $skip: Int!) {
    swaprDayDatas(
      first: 365
      skip: $skip
      where: { date_gt: $startTime }
      orderBy: date
      orderDirection: asc
    ) {
      date
      dailyVolumeUSD
      totalLiquidityUSD
    }
  }
`;
```